doc/code/converters/char_swap_attack_converter.py

```diff
@@ -25,7 +25,7 @@
 # %%
 from pyrit.common import IN_MEMORY, initialize_pyrit
 from pyrit.orchestrator import PromptSendingOrchestrator
-from pyrit.prompt_converter.charswap_attack_converter import CharSwapGenerator
+from pyrit.prompt_converter.charswap_attack_converter import CharSwapConverter
```

Comment:
```suggestion
from pyrit.prompt_converter import CharSwapConverter
```
nit: we prefer short imports